### PR TITLE
feat: added project tracking 

### PR DESCRIPTION
**Pull request by fpvyas**
created md file

### CODE REVIEW

File: docs/General/Meetings/project_tracking.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- **BHK review deadline**: Add the deadline for when you need Ben to review the task.
+- **Lab Members**: The lab members involved in this task. (If it's just yourself, only write your name. If it's multiple people, write all their names.)
+- **Comments**: Add any additional comments that you think are necessary.
+- **Relevant links**: Add any relevant links that are necessary for this task.
+
+
+### Manuscript Progress
+If you have any updates on the progress of your manuscript, [this spreadsheet](https://docs.google.com/spreadsheets/d/1dZcNEmIGCdFthwj5cskuATfDTJ-7Mup8D5gO_HrmIPE/edit?usp=sharing) is where you add your updates.
+The following are the columns in the spreadsheet:
+- **Lead Author (Mandatory)**: Add the name of the lead author of the manuscript. There can be multiple names if there are multiple lead authors working on this manuscript.
```
Comment (RuiyanNi) [inline docs/General/Meetings/project_tracking.md:25]: add a line before the bullet points to show them as bullet points on the website
```suggestion
The following are the columns in the spreadsheet:

- **Lead Author (Mandatory)**: Add the name of the lead author of the manuscript. There can be multiple names if there are multiple lead authors working on this manuscript.
```

File: docs/General/Meetings/project_tracking.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- **Paper (Mandatory)**: Add the title of the manuscript.
+- **URL to the Folder (Mandatory)**: Add the URL to the folder where the manuscript and relevant files, such as figures, are stored.
+- **Status**: A drop-down menu where you can select the status of the manuscript. For example, if the manuscript has been submitted, you can select 'Submitted to journal'
+- **Comments**: Add any additional comments that you think are necessary. This will include any updates on the progress of the manuscript, such as if there are experiments being done, any review necessary, etc. This will typically be updated during pre-lab meeting overview.
+
+
+### Grants & Award Applications
+Information and updates regarding any grants or awards the lab is applying for can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1EuFaUxtk2kPueZwmjTopyRvnFduuSbp5Byf0gEWA7kw/edit?usp=sharing). 
+The following are the columns in the spreadsheet:
+- **Proposed project/idea**: Add a general descriptive title for the project or idea.
```
Comment (RuiyanNi) [inline docs/General/Meetings/project_tracking.md:35]: add a new line before the bullet points
```suggestion
The following are the columns in the spreadsheet:

- **Proposed project/idea**: Add a general descriptive title for the project or idea.
```

File: docs/General/Meetings/project_tracking.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- **Deadline for Full Application**: Add the deadline for the full application submission.
+- **Status by Action**: Add the current status of the application using the drop-down menu.
+- **Link to the Folder**: Add the URL to the folder where the application and relevant files are stored.
+- **Comments**: Add any additional comments that you think are necessary.
+
+
+### Data Access Status
+Throughout your time in the lab, you may want to request access to additional datasets. [This spreadsheet](https://docs.google.com/spreadsheets/d/1j4umcD30PE8DM6LPCrkTUeKV3jUgaaPSUBis964FdrQ/edit?usp=sharing) is where you can track the status of your data access requests.
+The following are the columns in the spreadsheet:
+- **Dataset Name**: Add the name of the dataset you are requesting access to.
```
Comment (RuiyanNi) [inline docs/General/Meetings/project_tracking.md:54]: add a new line before the bullet points
```suggestion
The following are the columns in the spreadsheet:

- **Dataset Name**: Add the name of the dataset you are requesting access to.
```

**Review by RuiyanNi — CHANGES_REQUESTED** [review]
A tiny thing fixed. Otherwise, very comprehensive, great work!!!

**Review by RuiyanNi — APPROVED** [review]

**Review by mtran-code — APPROVED** [review]
Nice job 👍

**Pull request merged**